keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -107,6 +117,8 @@ serves the resourceVersion="0" list requests from reflectors today.
 
 ### Non-Goals
 
+- Remove all true quorum reads.
+- Serving pagination continuation from watch cache.
```

Comment:
nit: this just renders badly now with the [UNRESOLVED] below - can you fix this?

Reply:
Will try.

Reply:
Will remove it as in https://github.com/kubernetes/enhancements/pull/4047#discussion_r1226352953 now it's part of Beta milestone.